fictadvisor-api/src/v2/utils/documentation/pageText/updateByKey.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiDocumentationParams } from '../decorators';
```

Comment:
Name of the documentation file (updateByKey.ts) must be the same as corresponding controller method (updateMany.ts)